_data/chains/eip155-511.json

```diff
@@ -0,0 +1,24 @@
+{
+    "name": "OTC Mainnet",
+    "chain": "OTC",
+    "rpc": [
+      "https://test.otc.network"
+    ],
+    "faucets": [],  
+    "nativeCurrency": {
+      "name": "OTC",
+      "symbol": "OTC",
+      "decimals": 18
+    },
+    "infoURL": "https://www.otc.network/",
+    "shortName": "otc",
+    "chainId": 512,
+    "networkId": 512,
+    "slip44": 511,
+    "explorers": [{
+      "name": "otc",
+      "url": "https://explorer.otc.network",
```

Comment:
this URL cannot be reached

Reply:
ben-haim:master

Reply:
I will add soon the explorer @ligi thanks